### ISSUE TITLE
Less variable overrides are ignored

### ISSUE THREAD

**Issue by chrisnicola** [vwall/compass-twitter-bootstrap#20]
Here is my application.css.scss

```
@import "compass_twitter_bootstrap";
@import "compass_twitter_bootstrap_responsive";

@import "partials/base";
```

and in partials/_base.scss I have the line:

```
$baseFontSize:          16px;
```

As far as I understand it this should override the ridiculously small base font size bootstrap comes with by default, but it doesn't seem to have any effect. Any idea what I'm doing wrong?

**Comment by tmaier** [comment]
I guess we would need to load all variables with the `!default` option.
http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#variable_defaults_

You would need then to move your `_base` partial above `@import "compass_twitter_bootstrap";`

**Comment by chrisnicola** [comment]
Makes sense. Why did this work previously though? At least I'm pretty sure this was working prior to the 2.0 update.

**Comment by eugene-p** [comment]
To fix this issue you should change `_variables.scss`

`$baseFontSize: 13px;` >> `$baseFontSize: 13px !default;`

```
$baseFontSize: 16px;
@import "compass";
@import "compass_twitter_bootstrap";
```

Would set font size to 16px;

**Comment by vwall** [comment]
Fixed in d52b2fb06d9a2249353e7061f60be88557115962
